portfolio/src/main/webapp/script.js

```diff
@@ -56,32 +56,27 @@ function deleteComments() {
   });
 }
 
-var slideIndex = 0;
+var slideIndex = [1,1];
+var slideId = ["slides-fade1", "slides-fade2"]
+showSlides(1, 0);
+showSlides(1, 1);
 
-// Next/previous controls
-function plusSlides(n) {
-  showSlides(slideIndex += n);
+function plusSlides(n, no) {
+  showSlides(slideIndex[no] += n, no);
 }
 
-// Thumbnail image controls
-function currentSlide(n) {
-  showSlides(slideIndex = n);
-}
-
-// Add it to the page.
-const slideContainer = document.getElementById('slides-fade');
-
-function showSlides(n) {
+function showSlides(n, no) {
   var i;
-  var slides = document.getElementsByClassName('slides-fade');
-  if (n > slides.length) {slideIndex = 0}
-  if (n < 0) {slideIndex = slides.length}
-  for (i = 0; i < slides.length; i++) {
-      slides[i].style.display = 'none';
+  var x = document.getElementsByClassName(slideId[no]);
+  if (n > x.length) {slideIndex[no] = 0}    
+  if (n < 0) {slideIndex[no] = x.length}
```

Comment:
I am no expert on javascript style. But it's good to keep it consistent with the style shown in line 73-77 in the for loop.